client/components/tinymce/plugins/simple-payments/plugin.jsx

```diff
@@ -9,16 +9,15 @@ import { unmountComponentAtNode } from 'react-dom';
  * Internal Dependencies
  */
 import SimplePaymentsDialog from './dialog';
+import { serialize } from './shortcode-utils';
 import { renderWithReduxStore } from 'lib/react-helpers';
 
 const simplePayments = editor => {
 	let node;
 	const store = editor.getParam( 'redux_store' );
 
 	editor.on( 'init', () => {
-		node = editor.getContainer().appendChild(
-			document.createElement( 'div' )
-		);
+		node = editor.getContainer().appendChild( document.createElement( 'div' ) );
```

Comment:
I use Prettier to format code -- and it sometimes changes a perfectly valid and nice formatting into another valid and nice one :)

Would it help if I moved the formatting-only changes to a separate commit?